documentation/docs/03-template-syntax/11-bind.md

```diff
@@ -53,6 +53,19 @@ In the case of a numeric input (`type="number"` or `type="range"`), the value wi
 
 If the input is empty or invalid (in the case of `type="number"`), the value is `undefined`.
 
+You can give the input a default value by setting the `defaultValue` property. This way, when the input is part of a form and its `form.reset()` method is invoked, it will revert to that value instead of the empty string. Note that for the initial render the value of the binding takes precedence if it's not `null` or `undefined`.
+
+```svelte
+<script>
+	let value = $state('');
+</script>
+
+<form>
+	<input bind:value defaultValue="x">
+	<input type="reset" value="Reset">
+</form>
+```
```

Comment:
MDN [has a note about default values](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/reset) which I think it worth reiterating here

````suggestion
```

> [!NOTE]
> Use reset buttons sparingly, and ensure that users won't accidentally click them while trying to submit the form.
````